mkdocs.yml

```diff
@@ -86,6 +86,8 @@ nav:
           - GAMS:       Documentation/Applications/gams.md
           - Gaussian:   Documentation/Applications/gaussian.md
           - Gurobi:     Documentation/Applications/gurobi.md
+          - IDAES Solvers : Documentation/Applications/idaes_solvers.md
+          - IPOPT:      Documentation/Applications/ipopt.md
```

Comment:
IPOPT is still listed in the Libraries section too, that needs to be removed in mkdocs.yml. 

Would it make sense to move HSL to this section too? It seems like it makes sense to have the IPOPT page and HSL page together

Reply:
HSL is technically a library, even Ipopt uses its `libcoinhsl.so` for solving the system of equations. We can move it to Applications, but it's very hard to use it as a standalone app. However, if you think that's a better place for it, I can move it in a subsequent PR.

Reply:
Ok, whichever you think makes the most sense.